project/Dependencies.scala

```diff
@@ -126,10 +128,17 @@ object Dependencies {
   val googleCloudNio: ModuleID =  "com.google.cloud"    % "google-cloud-nio"      % "0.124.21" % Test // brought in for FakeStorageInterpreter
 
   val circeYaml =         "io.circe"          %% "circe-yaml"           % "0.14.2"
+  val circeCore =         "io.circe"          %% "circe-core"           % "0.14.3"
   val http4sBlazeServer = "org.http4s"        %% "http4s-blaze-server"  % http4sVersion
   val http4sPrometheus = "org.http4s" %% "http4s-prometheus-metrics" % http4sVersion
   val http4sDsl =         "org.http4s"        %% "http4s-dsl"           % http4sVersion
+  val http4sEmberClient = "org.http4s"        %% "http4s-ember-client"  % http4sVersion
```

Comment:
can we use the default http client instead of ember client?

Reply:
By default client, I am assuming you mean the client used here [HttpDockerDAOSpec.scala#L68](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpDockerDAOSpec.scala#L68). I have updated the code accordingly. If that's not what you meant please let me know. The reason I use Ember instead of Blaze is because the latter has been deprecated.